drop-cap/library/src/main/java/com/novoda/dropcap/DropCapView.java

```diff
@@ -246,49 +251,61 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
         super.onMeasure(widthMeasureSpec, desiredHeightMeasureSpec);
     }
 
-    private void measureDropCapFor(int width) {
-        dropCapWidth = (int) (dropCapPaint.measureText(dropCapText, 0, 1) + spacer);
-        dropCapPaint.getTextBounds(dropCapText, 0, 1, dropCapBounds);
-        int copyWidthForDropCap = width - dropCapWidth;
+    private void measureDropCapFor(int totalWidth) {
+        dropCapWidth = (int) (dropCapPaint.measureText(dropCapText, 0, dropCapText.length()) + spacer);
+        dropCapPaint.getTextBounds(dropCapText, 0, dropCapText.length(), dropCapBounds);
+        int remainingWidthAfterDropCap = totalWidth - dropCapWidth;
 
-        if (dropCapStaticLayout == null || dropCapStaticLayout.getWidth() != copyWidthForDropCap) {
-            dropCapStaticLayout = new StaticLayout(
-                    dropCapText + copyText,
+        if (dropCapCopyStaticLayout == null || dropCapCopyStaticLayout.getWidth() != remainingWidthAfterDropCap) {
+            dropCapCopyStaticLayout = new StaticLayout(
+                    copyText,
```

Comment:
Unnecessarily relying on the `DropCapText` and `copyText` when this only deals with the remaining text, the whole text minus the `DropCap` text. 

This makes the drawing process easier for the copy wrapping the `DropCap` because it does not need to check the start position and offset it by the `DropCapText.length()`.